addons/strings/script_component.hpp

```diff
@@ -10,3 +10,222 @@
 #endif
 
 #include "\x\cba\addons\main\script_macros.hpp"
+
+#define UTF8_TABLE [\
+    ["%21","!"],\
```

Comment:
These are just ascii keycodes
https://community.bistudio.com/wiki/toString

Reply:
You have any good way to translate i.e. `%e2%80%98"` using `toString`?

Reply:
Tested this:
```
missionName // "%c2%ac%c2%ae%c2%af%c2%b0%c2%b1%c2%b2%c2%b3%c2%b4%c2%b5%c2%b6%c2%b7%c2%b8%c2%b9%c2%ba%c2%bb%c2%bc%c2%bd%c2%be%c2%bf"

missionName call CBA_fnc_decodeURL // "¬®¯°±²³´µ¶·¸¹º»¼½¾¿"

toString (missionName splitString "%" apply {call compile format ["0x%1", _x]} apply { _x}) // "Â¬Â®Â¯Â°Â±Â²Â³Â´ÂµÂ¶Â·Â¸Â¹ÂºÂ»Â¼Â½Â¾Â¿"
```

So it is not exactly the same as ASCII codes. It seems to follow the UTF-8 conversion table instead:
https://www.w3schools.com/tags/ref_urlencode.asp
Except that % is encoded as %% and not %25.